zookeeper-server/src/main/java/org/apache/zookeeper/Op.java

```diff
@@ -152,6 +152,45 @@ public static Op create(String path, byte[] data, List<ACL> acl, CreateMode crea
         return new Create(path, data, acl, createMode);
     }
 
+    /**
+     * Constructs a create operation which uses given op code if no one is inferred from create mode.
+     *
+     * @param path
+     *                the path for the node
+     * @param data
+     *                the initial data for the node
+     * @param options
+     *                options for creating znode
+     * @param defaultOpCode
+     *                op code to be used if no one is inferred from create mode
+     */
+    static Op create(String path, byte[] data, CreateOptions options, int defaultOpCode) {
+        if (options.getCreateMode().isTTL()) {
```

Comment:
it seems to me that we are not exercising the code paths with TTL in the new tests

Reply:
Amend `CreateTTLTest.testMulti` for this.